ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoBackend.scala

```diff
@@ -44,6 +44,7 @@ private[dao] trait JdbcLedgerDaoBackend extends AkkaBeforeAndAfterAll {
     JdbcLedgerDao.writeOwner(
       serverRole = ServerRole.Testing(getClass),
       jdbcUrl = jdbcUrl,
+      connectionPoolSize = 16,
```

Comment:
Has this number any specific importance for the tests and why (or why not)?

Reply:
This used to be the previous default and I didn't want to change the behavior of tests. I will put a comment there.